modules/social_features/social_event/modules/social_event_type/social_event_type.module

```diff
@@ -64,29 +64,32 @@ function social_event_type_widget_alter(array &$form) {
  *   Array with variables of a node.
  */
 function social_event_type_preprocess_node(array &$variables) {
-  /* @var \Drupal\node\NodeInterface $node */
+  /** @var \Drupal\node\Entity\NodeInterface $node */
   $node = $variables['node'];
+
   if ($node->getType() === 'event') {
     $event_type = $node->get('field_event_type');
     $event_type_entities = $event_type->referencedEntities();
     if (count($event_type_entities) === 1) {
       /** @var \Drupal\node\Entity\Node $event */
       foreach ($event_type_entities as $event) {
-        $variables['metadata'] = t('in @event', ['@event' => $event->link()]);
-        // Set event type link.
-        $now = strtotime("now");
-        $end = strtotime($node->get('field_event_date_end')->getString());
-        // 1 are upcoming events.
-        // 2 are ones that have started or are finished.
-        $datetype = ($now > $end) ? 2 : 1;
-
-        $event_type_url = Url::fromRoute('view.upcoming_events.page_community_events', [
-          'field_event_date_value' => $datetype,
-          'event_type_id' => $event->id(),
-        ]);
-        $event_type_link = Link::fromTextAndUrl($event->label(), $event_type_url)
-          ->toString();
-        $variables['event_type'] = $event_type_link;
+        if ($event instanceof Node) {
```

Comment:
```suggestion
        if ($event instanceof NodeInterface) {
```

Shouldn't this be `NodeInterface`?

Reply:
Perhaps, but looking at the `/** @var \Drupal\node\Entity\Node $event */` which was already existing before this release that didn't really make sense. Or that should also be removed.